### PR TITLE
Id 544

### PR DESCRIPTION
**Pull request by TKhardekar**
#### 5 markdown files common to both XLD and XLR

1. customization.md (quite a bit of restructuring, paraphrasing and typo correction and use of US spelling:  customize)

2. environment-variables.md (quite a bit of paraphrasing involved as some sentences did not convey the correct intention, corrected sentences that used "..." instead of a conjunction, checked for case inconsistency)

3. single-node-deployment.md (some paraphrasing involved)

4. upgrade-single-node-deployment.md (some paraphrasing involved, some grammar and typo correction involved)

5. volumes.md (quite a bit of paraphrasing required)

#### XL_Deploy specific markdown files

1. update-multi-node-deployment.md (some paraphrasing, grammar checking involved)

1. Readme.md (paraphrasing and typo checking done)
Added links to XL UP

#### XL_Release specific files

1. update-multi-node-deployment.md (some paraphrasing, grammar checking involved)

1. Readme.md (paraphrasing and typo checking done)
Added links to XL UP

### CODE REVIEW

File: docs/customisation.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+**Note:** Certain JARS should be placed in specific paths only. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team.
 
-Please note that certain JARS will go into certain paths. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team. 
+**Note:** `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
 
-NOTE: `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
+To begin, create a `Dockerfile` that resembles the following configuration:
 
-```docker
+```
```
Comment (deepu105) [inline docs/customisation.md]: this is incorrect, should be on beginning like ` ```docker ` as it was to use correct syntax highlight

File: docs/customisation.md [hunk 2]
```diff
@@ -51,40 +52,45 @@ ADD --chown=10001:0 files/sattelite-lib-hotfix.jar /opt/xebialabs/xl-deploy-serv
...
 ## XL Release
 
-As with XL Deploy, always remember that the owning user:group combination _must_ be `10001:0` for any files you add.
+**Important:** When you create a Dockerfile with custom resources added, always remember that the owning user:group combination _must_ be `10001:0`.
 
-To start off with, create a `Dockerfile` that resembles the following. 
+**Note:** Certain JARS should be placed in specific paths only. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team.
 
-Please note that certain JARS will go into certain paths. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team. 
+**Note:** `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
```
Comment (deepu105) [inline docs/customisation.md]:
```suggestion
**Note:** `${APP_HOME}` points to the path `/opt/xebialabs/xl-release-server` by default
```

File: docs/environment-variables.md [hunk 3]
```diff
@@ -1,116 +1,116 @@
...
 - default value: content of `${APP_HOME}/conf/repository-keystore.jceks` which is generated from `keytool -genseckey -alias deployit-passsword-key -keyalg aes -keysize 128 -keypass "deployit" -keystore ${APP_HOME}/conf/repository-keystore.jceks -storetype jceks -storepass ${REPOSITORY_KEYSTORE_PASSPHRASE}`
 - example: "b3J0IHBvbGljeTogMSBtb250aCBvZiB0ZWNobmljYWwgc3VwcG9ydApFZGl0aW9uOiBUcmlhbAotLS0gU2lnbmF0dXJlIChTSEExd2l0aERTQSkgLS0tCjMwMmMwM"
-- Note:    
+- **Note:**    
     - This value is set as string whihc will be converted to base64 value and added to keystore file like below,
 `echo "${REPOSITORY_KEYSTORE}" | base64 -d > ${APP_HOME}/conf/repository-keystore.jceks`
 
 ##### `XL_LICENSE`
-- Sets your XL License by passing base64 string license .. which will be added to license file .. other options to pass it as volume check Volumes docs for more details.
+- Sets your XL License by passing a base64 string license, which will then be added to license file. For more details on other options to pass the license as a volume, see [Volumes](##).
```
Comment (deepu105) [inline docs/environment-variables.md]:
```suggestion
- Sets your XL License by passing a base64 string license, which will then be added to the license file. For more details on other options to pass the license in a volume, see [Volumes](##).
```

File: docs/environment-variables.md [hunk 4]
```diff
@@ -1,116 +1,116 @@
...
 - possible values: "license-base64-value"
 - default value: unregistered license valid for 7 days but you must use `ACCEPT_EULA`.
 - example: "LS0tIExpY2Vuc2UgLS0tCkxpY2Vuc2UgdmVyc2lvbjogMwpQcm9kdWN0OiOiBUcmlhbAotLS0gU2lnbmF0dXJlIChTSEExd2l0aERTQSkgLS0tCjMwMmMwMjE0MTY5YjUyNGFkYWYwYWRhYmNlZDcyMGM0YmZkMGmVhNTZhNGFiMzk4YTgzNGUyZjJiOWQ1CjI2MzNhHVyZSAtLS0K%"
 
 ##### `ACCEPT_EULA`
-- This is needed "you must accept the End User License Agreement" if you will not provide your own license before container can start.
+-"You must accept the End User License Agreement" if are unable to provide your own license before container can start.
```
Comment (deepu105) [inline docs/environment-variables.md]:
```suggestion
-"You must accept the End User License Agreement" if you are unable to provide your own license before the container can start.
```

File: docs/environment-variables.md [hunk 5]
```diff
@@ -1,116 +1,116 @@
...
-- Force upgrade setting. In case of upgrade it will be performed in non-interactive mode by passing flag `-force-upgrades` during service starting.
-- possible values: "true", "false" 
-- default value: "false" 
+- Force upgrade setting. It can be used to perform an upgrade in non-interactive mode by passing flag `-force-upgrades` while  starting a service.
+- possible values: "true", "false"
+- default value: "false"
 
 ##### `XL_CLUSTER_MODE`
-- This is to specify if HA setup needed and specify which mode.
+- This is to specify if HA setup is needed and specify modes of setup.
```
Comment (deepu105) [inline docs/environment-variables.md]:
```suggestion
- This is to specify if the HA setup is needed and to specify the HA mode.
```

File: docs/environment-variables.md [hunk 6]
```diff
@@ -1,116 +1,116 @@
...
 ##### `XL_DB_PASSWORD`
-- Password used to connect to DataBase configured with XL products.
+- Password used to connect to database configured with XL products.
 - possible values: "password-value"
 - default value: "123"
 - example: "xl_P@ssw0rd01"
 
 ##### `XL_METRICS_ENABLED`
-- Flag to exposes internal and system metrics over Java Management Extensions (JMX). so that any monitoring system that can read JMX data can be used to monitor XL Product.
+- Flag to expose internal and system metrics over Java Management Extensions (JMX). This is to enable the use of a monitoring systems that can read JMX data, with XL Products.
```
Comment (deepu105) [inline docs/environment-variables.md]:
```suggestion
- Flag to expose internal and system metrics over Java Management Extensions (JMX). This is to enable the use of monitoring systems that can read JMX data, with XL Products.
```

File: docs/customisation.md (hunk 1 above)
Comment (TKhardekar) [reply docs/customisation.md]: Understood, will change it. Thanks.

**Pull request merged**